syz-manager/html.go

```diff
@@ -309,6 +310,23 @@ func (mgr *Manager) httpCoverCover(w http.ResponseWriter, r *http.Request, funcF
 	}
 	mgr.mu.Unlock()
 
+	// On arm64 as PLT is enabled by default, .text section is loaded after .plt section,
+	// so there is 0x18 bytes offset from module load address for .text section
+	// we need to remove the 0x18 bytes offset in order to correct module symbol address
+	if mgr.target.Arch == targets.ARM64 {
+		for _, prog := range progs {
+			var nPCs []uint64
+			for _, pc := range prog.PCs {
+				// TODO: avoid to hardcode the address
+				if pc < 0xffffffd010000000 {
```

Comment:
It feels wrong that these lowest level details are added to the highest level program. We bother manager with with some arch-specific magic constants. This is also duplicated 3 times and makes the logic in pkg/cover untestable: pkg/cover will fail on end-to-end tests and this piece is untestable. Also other users of pkg/cover miss this, e.g. tools/syz-cover.
Please find a proper place for this somewhere under pkg/cover.